Formula/freedink.rb

```diff
@@ -34,8 +34,7 @@ def install
   end
 
   test do
-    assert FileTest.executable?("#{bin}/freedink")
-    assert shell_output("#{bin}/freedink -vwis").split("\n").first == "GNU FreeDink 108.4"
+    assert_match "GNU FreeDink 108.4", shell_output("#{bin}/freedink -vwis").split("\n").first
```

Comment:
You can remove `.split("\n").first`; `assert_match` already ensures `GNU FreeDink 108.4` is *somewhere* in that output. We have `assert_equal` for strict equality.